### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by zielvna**

### CODE REVIEW

File: test/clamm.test.ts [hunk 1]
```diff
@@ -1534,6 +1559,20 @@ describe('math tests', () => {
       }
       await expectError(clamm.methods.calculateAmountDelta(params))
     })
+
+    test('error handling 2', async () => {
```
Comment (Sniezka1927) [inline test/clamm.test.ts:1563]: The test names should be more precise, you should mention which error the cases handle

File: test/clamm.test.ts [hunk 2]
```diff
@@ -710,7 +710,7 @@ describe('math tests', () => {
     const minSqrtPriceOutsideDomain = 1n
     const minLiquidity = 1n
     const maxLiquidity = (1n << 256n) - 1n
-    const minOverflowTokenY = 340282366920940n
+    const minOverflowTokenY = 115792089237316195423570985008687907853269984665575031n
```
Comment (Sniezka1927) [inline test/clamm.test.ts:713]: From where the diffrence come from?

Comment (zielvna) [reply test/clamm.test.ts:713]: `SqrtPrice` type on Casper is `u128` and `SqrtPrice` on Alephium is `U256` that's why required number of tokens to cause overflow is higher

File: test/uints.test.ts [hunk 3]
```diff
@@ -166,6 +166,37 @@ describe('uints tests', () => {
...
+    }
+  })
+
+  test('big add 256 vs new big add 256 comparison', async () => {
+    {
+      const a = MaxU256
+      const b = MaxU256
+      const oldResult = await uints.methods.bigAdd256({ args: { a, b } })
+      const newResult = await uints.methods.newBigAdd256({ args: { a, b } })
+      console.log('big add 256', oldResult.gasUsed, newResult.gasUsed)
```
Comment (Sniezka1927) [inline test/uints.test.ts:196]: I think assertion that the new methods use less gas will be better than just console logging it. Fix it in all other cases too.
Even if they are using less gas it is worth to consider if we still need previous implementations.

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**